guide/daily-spending-wallet/requesting.md

```diff
@@ -34,439 +34,454 @@ Illustration source
-
-For some background on situations the user might encounter with different invoice expiries, see this [blog post](https://d.elor.me/2022/01/lightning-invoice-expiration-ux-considerations/) from designer Stephen DeLorme.
-
-Using [BOLT12](https://bolt12.org/) offers, invoice expirations will no longer be a point of user friction. Offers are not yet an accepted standard, or widely supported by wallets, so interoperability will be an issue if adopting it.
-
-</div>
-
-### Address types
-
-For on-chain, there are several address types that can be used for single-use payment requests.
-
-<div class="center" markdown="1">
-
 {% include picture.html
-   image = "/assets/images/guide/daily-spending-wallet/requesting/single-use-payment-request-settings-address.png"
-   retina = "/assets/images/guide/daily-spending-wallet/requesting/single-use-payment-request-settings-address@2x.png"
-   layout = "float-right-desktop -background -shadow"
-   caption = "Supporting every address type benefits users and makes your wallet more interoperable."
-   alt-text = "Screen showing different address types."
+   image = "/assets/images/guide/daily-spending-wallet/requesting/Invoice.png"
+   retina = "/assets/images/guide/daily-spending-wallet/requesting/Invoice@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/requesting/Invoice@2x.png"
+   layout = "shadow"
+   caption = "Let users share the lightning invoice independently."
+   alt-text = "Screen showing an invoice that is part of a unified request in QR format."
    width = 250
    height = 541
+   modalWidth = 250
+   modalHeight = 541
 %}
 
-Each address type has unique benefits, such as SegWit saving on network fees and Taproot offering Lightning users increased privacy.
-
-You should encourage users to use the latest, more feature rich address types. Having an option in the payment request settings to set which address type to be used by default is recommended.
-
-Older address types should always be supported for backwards compatibility with older wallets.
-
-More on the [address types]({{ "/guide/glossary/address/" | relative_url }}) and [interoperability]({{ "/guide/designing-products/interoperability/" | relative_url }}) page.
-
-</div>
-
-### Metadata
-
-A payment request with only standard data, like an amount and date, communicates little to both parties about purpose and context of the payment.
-
-<div class="center" markdown="1">
-
 {% include picture.html
-   image = "/assets/images/guide/daily-spending-wallet/requesting/single-use-payment-request-settings.png"
-   retina = "/assets/images/guide/daily-spending-wallet/requesting/single-use-payment-request-settings@2x.png"
-   layout = "float-left-desktop -background -shadow"
-   caption = "Let users attach payment metadata to single-use payment requests."
-   alt-text = "Screen showing single payment request settings."
+   image = "/assets/images/guide/daily-spending-wallet/requesting/address.png"
```

Comment:
Not clear how this screen is so different from above? Shouldn't all these fields be also shown further up the page?

`Name` is probably not needed. That's what you would handle via contact selection.

Reply:
Name is the requesters name which would use something like NameDesc with BOLT 11 invoices or with BOLT 12 offers there is a name field that acts the same, can remove though